### PR TITLE
Allow disabling gestures

### PR DESCRIPTION
**Pull request by Mugurell**
Also prelands the Strings needed for https://github.com/mozilla-mobile/fenix/issues/11862

### Pull Request checklist

- [x] **Tests**: No tests. Small UI changes.
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made.
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md). No suggestions from the `Accessibility Scanner` for the "Customize" settings screen.

[video showing the changes](https://drive.google.com/file/d/1mtUv3mPFPDbaHxcM48n6y2R4Jh4UD9fg/view?usp=sharing)

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -451,6 +453,17 @@
     <!-- Preference for using following device theme -->
     <string name="preference_follow_device_theme">Follow device theme</string>
 
+    <!-- Gestures Preferences-->
+    <string name="preference_gestures_group_name">Gestures</string>
```
Comment (mcarare) [inline app/src/main/res/values/strings.xml:457]: Is `preference_gestures_group_name` used anywhere? It seems preferences_gestures is defined for preference category name.

**Review by mcarare — APPROVED** [review]
LGTM, with just the string preference question!

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Mugurell) [reply app/src/main/res/values/strings.xml]: Eagle eye! Thanks!
Wasn't sure where to put the name, how to name the string.
Seems like `preferences_gestures` won and is more in line with the titles of the other categories.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -282,6 +282,8 @@
     <string name="preferences_theme">Theme</string>
     <!-- Preference for customizing the home screen -->
     <string name="preferences_home">Home</string>
+    <!-- Preference for gestures based actions -->
+    <string name="preferences_gestures">Gestures</string>
```
Comment (ekager) [inline app/src/main/res/values/strings.xml:286]: @betsymi could you confirm these strings? :)

Comment (betsymi) [reply app/src/main/res/values/strings.xml:286]: Looks good.

**Pull request merged**